<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>makeAutoPlotMulti</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose a CM, a group of signals, Striptool or ArchiveViewer, then click Go!</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cavity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cryomodule</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Start ST or AV:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="CMComboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="CavComboBox"/>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="STradioButton">
           <property name="text">
            <string>StripTool</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="AVradioButton">
           <property name="text">
            <string>ArchiveViewer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="GoButton">
         <property name="text">
          <string>Go!</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="UsualPB">
         <property name="text">
          <string>Usual Commissioning Suspects</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearPB">
         <property name="text">
          <string>Clear Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="CheckBoxGrid"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
